.pipelines/pipeline.yaml

```
@@ -14,7 +14,7 @@ pr:
 trigger:
   branches:
     include:
-      - gh-readonly-queue/master/*
+      - gh-readonly-queue/release/v1.5/*
```

maybe we should preemptively go
```suggestion
      - gh-readonly-queue/release/*
```
so that we don't have to add to this every few months?

---

Should this target master instead and then backport?

---

We can take this and make that change an master also